cilium/cmd/bpf_ct_list.go

```diff
@@ -34,13 +34,8 @@ var (
 				cmd.PrintErrf("Invalid argument: %s", err.Error())
 				return
 			}
-			maps := getMaps(t, id)
-			ctMaps := make([]interface{}, len(maps))
-			for i, m := range maps {
-				ctMaps[i] = m
-			}
 			common.RequireRootPrivilege("cilium bpf ct list")
-			dumpCt(ctMaps, t)
+			dumpCt(mapsToInterfaceSlice(getMaps(t, id)), t)
```

Comment:
Can't we fix up `dumpCt` a bit to make it typed? This empty interface business isn't ideal.